numpy/core/fromnumeric.py

```diff
@@ -1764,9 +1764,9 @@ def nonzero(a):
     Returns a tuple of arrays, one for each dimension of `a`,
     containing the indices of the non-zero elements in that
     dimension. The values in `a` are always tested and returned in
-    row-major, C-style order. The corresponding non-zero
+    row-major, C-style order. The corresponding non-zero  
     values can be obtained with::
-
+    
```

Comment:
Nit: let's not add random whitespace to the patch

Reply:
Sorry, I was changing from my phone.